scrapper.py

```diff
@@ -83,9 +109,31 @@ def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path, 'r') as crawler:
+        data = json.loads(str(crawler.read()))
```

Comment:
we will cover that next lecture, but there is another method json.load that allows to read json file from the opened file descriptor. read more in docs. current way is working, but is more a workaround and will not be accepted